src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,55 @@
+# Install and configure Chocolatey
+# See: https://community.chocolatey.org/
+
+function Install-Chocolatey {
+    # Download and execute the Chocolatey install script
+    iex ((New-Object System.Net.WebClient).DownloadString("https://community.chocolatey.org/install.ps1"))
+}
+
+function Set-ChocoConfig {
+    # Configure Chocolatey to reference packages from Mandiant's VM-Packages repository
+    # See: https://github.com/mandiant/VM-Packages
+    iex "choco sources add --name vm-packages --source 'https://www.myget.org/F/vm-packages/api/v2;https://myget.org/F/vm-packages/api/v2' --priority 1"
+    iex "choco feature enable --name allowGlobalConfirmation"
+    iex "choco feature enable --name allowEmptyChecksums"
```

Comment:
In line with https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/91aba6599d7aca0071a30f2abac242c926fb47d1 are these `iex` calls necessary?

Reply:
@mcdonnnj resolved